ontology/omrse.md

```diff
@@ -27,6 +27,10 @@ build:
 activity_status: active
 repository: https://github.com/ufbmi/OMRSE
 preferredPrefix: OMRSE
+publications:
+  - id: https://jbiomedsem.biomedcentral.com/articles/10.1186/s13326-016-0087-8
```

Comment:
Please use the PubMed identifier since a URL to a journal isn't really actionable
```suggestion
  - id: https://www.ncbi.nlm.nih.gov/pubmed/27406187
```